hello_world_print.py

```diff
@@ -1,3 +1,15 @@
 # A program to print hello world using the print function
 
 print('Hello, World!')
```

Comment:
Since the name of the original file isn't as per the repository norms, you can also fix that.

Change the original name from 'hello_world_print.py' to 'using-print.py'